### PR TITLE
Fix builds with --without-bugzilla

### PR DESCRIPTION
**Pull request by michalfabik**
Fix various issues related to conditional building with the `--without-bugzilla` `./configure` option, esp. in non-Fedora distributions.

Resolves: https://github.com/abrt/libreport/issues/755

### CODE REVIEW

**Comment by msrb** [comment]
+1 :wink:

**Comment by michalfabik** [comment]
@msrb , @mgrabovsky , please have a quick look at the commits I added today. I'll make a new release after that.

File: Makefile.am [hunk 1]
```diff
@@ -4,9 +4,8 @@ SUBDIRS = data po src tests doc apidoc
 EXTRA_DIST = \
     libreport.pc.in \
     libreport-version \
-    asciidoc.conf
-
-dist_doc_DATA = README.md
+    asciidoc.conf \
+	README.md
```
Comment (mgrabovsky) [inline Makefile.am:8]: Tab here should be 4 spaces to maintain consistency.

Comment (michalfabik) [reply Makefile.am]: Thanks, fixed.

**Review by mgrabovsky — APPROVED** [review]
Looks fine to me.

**Pull request merged**